packages/core/database/lib/entity-manager/index.js

```diff
@@ -364,17 +397,19 @@ const createEntityManager = (db) => {
             // set columns
             const { idColumn, typeColumn } = targetAttribute.morphColumn;
 
+            const relId = toId(cleanRelationData.set[0]);
```

Comment:
We should check if cleanRelationData is null here.

Reply:
done :)